virtme/commands/configkernel.py

```diff
@@ -365,7 +365,8 @@ def do_it():
 
     # Check if KBUILD_OUTPUT is defined and if it's a directory
     config_dir = os.environ.get("KBUILD_OUTPUT")
-    if config_dir is not None and os.path.isdir(config_dir):
+    if config_dir is not None:
+        os.makedirs(config_dir, exist_ok=True)
```

Comment:
Maybe catch if makedirs fails and report an error, something like:
```
try:
    os.makedirs(config_dir, exist_ok=True)
except:
    print("Error: invalid directory for KBUILD_OUTPUT")
    return 1
```

Reply:
Good idea! Done!

Reply:
And now Pylint is happy.